.github/workflows/docs-ci.yaml

```diff
@@ -26,9 +26,13 @@ jobs:
       with:
         repository: ${{ github.repository_owner }}/network-operator-docs  # repo must be explicitly set here for workflow calling to behave correctly
         token: ${{ inputs.token || secrets.GH_TOKEN_NVIDIA_CI_CD }}  # token must be explicitly set here for push to work in following step
+    - name: Setup Go
+      uses: actions/setup-go@0a12ed9d6a96ab950c8f026ed9f722fe0da7ef32 # v5.0.2
```

Comment:
Are the Make targets sensitive to the version of Go?
If not:
- Then the GitHub runner's workspace already has Go available, so this step is redundant.

If yes:
- Prefer to use [an explicit tag (version) of the action](https://github.com/actions/setup-go/tags) rather than a commit hash...
- Furthermore, who is responsible to increment this specific version in the not-near future? perhaps there is a solution that could improve the Go code itself to be less version-sensitive?

Reply:
So the bug is related to `make api-docs` which rely on a go package. It run successfully locally with GO 1.22.7/8 and go 1.23. So my supposition is that it's related to the go installation. I hope that a fresh install can fix it.

Reply:
What about adding a dependency for the package (and its specific version) in the code?

Reply:
It is already done.